### ISSUE TITLE
Add 996ICU license

### ISSUE THREAD

**Issue by MartinPetkov** [nexB/scancode-toolkit#1486]
### Description
Add the newly created 996ICU license: https://raw.githubusercontent.com/996icu/996.ICU/7b5aa09/LICENSE

Main repo: https://github.com/996icu/996.ICU

FYI it says it's meant to be used in combination with other licenses, which makes it less of a license on its own and more of a modifier.

**Comment by pombredanne** [comment]
@MartinPetkov Thank you! Do you know if this is worth adding right away or shall we wait a little since this seems to be a draft at this stage per https://github.com/996icu/996.ICU/blob/a5cf6d7d71e674a356eecffc1a6c60c199e72f66/LICENSE#L3

**Comment by MartinPetkov** [comment]
I think it's fine to wait, just wanted to have the issue open since I came across this on Hacker News.

**Comment by pombredanne** [comment]
@MartinPetkov yes and I really appreciate this as the more licenses the better. And this is also an ethical license (though weirdly enough its anti slave labor provisions could make it a non "true" open source license)

**Comment by pombredanne** [comment]
> as the more licenses the better.

I meant the better the license detection is ... (I kinda hate that tere are so many licenses otherwise ;) )
